CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to React Native ART
+
+## Development Process
+
+All work on React Native ART happens directly on GitHub. Contributors send pull requests which go through review process.
+
+> **Working on your first pull request?** You can learn how from this _free_ series: [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+1. Fork the repo and create your branch from `master` (a guide on [how to fork a repository](https://help.github.com/articles/fork-a-repo/)).
+2. Run `yarn` or `npm install` to install all required dependencies.
+3. Now you are ready to do the changes.
+
+## Testing your changes
+
+You can test your changes by installing `example` app on a simulator or device:
+
+Installing on a iOS:
+
+- Open `example/ios/example.xcodeproj`
+- Click `run` in the top left corner of `Xcode`.
```

Comment:
or use `yarn react-native run-ios`?

Reply:
It wasn't quite working for me.

Reply:
why :< would be nice to figure it out and fix

Reply:
True, I'll create an issue for that